server/models/Event.js

```diff
@@ -1,12 +1,25 @@
 const mongoose = require('mongoose');
+
+const pointSchema = new mongoose.Schema({
+  type: {
+    type: String,
+    enum: ['Point'],
+    required: true,
```

Comment:
Not sure the purpose here, enum with one item?

Reply:
I took it from the mongoose docs, I think [GeoJSON](https://mongoosejs.com/docs/geojson.html) points needs to be in this format in order to work